### PR TITLE
libxtst, bump revision after changes to util-macros and xorgproto

### PR DESCRIPTION
**Pull request by Begasus**

### CODE REVIEW

File: x11-libs/libxtst/libxtst-1.2.3.recipe [hunk 1]
```diff
@@ -5,62 +5,79 @@ HOMEPAGE="http://xorg.freedesktop.org/releases/individual/lib/"
...
-	devel:util_macros$secondaryArchSuffix
-	cmd:autoconf
-	cmd:libtool
 	cmd:aclocal
-	cmd:make
+	cmd:autoconf
+	cmd:awk
 	cmd:gcc$secondaryArchSuffix
-	cmd:ld$secondaryArchSuffix
+	cmd:libtool$secondaryArchSuffix
```
Comment (fbrosson) [inline x11-libs/libxtst/libxtst-1.2.3.recipe]: I think we prefer this variant:
```diff
-	cmd:libtool$secondaryArchSuffix
+	cmd:libtoolize$secondaryArchSuffix
```

Comment (Begasus) [reply x11-libs/libxtst/libxtst-1.2.3.recipe]: Thanks, missed this one ;)

File: x11-libs/libxtst/libxtst-1.2.3.recipe [hunk 2]
```diff
@@ -5,62 +5,79 @@ HOMEPAGE="http://xorg.freedesktop.org/releases/individual/lib/"
...
 	"
 
+defineDebugInfoPackage libxtst$secondaryArchSuffix \
+	$libDir/libXtst.so.$libVersion
+
 BUILD()
 {
-	libtoolize --force --copy --install
-	autoreconf -i
+	autoreconf -vfi
```
Comment (alaviss) [inline x11-libs/libxtst/libxtst-1.2.3.recipe]: I don't think this is necessary.

Comment (Begasus) [reply x11-libs/libxtst/libxtst-1.2.3.recipe]: I think there are many cases these can be dropped ...

**Comment by Begasus** [comment]
@alaviss nothing happens with the TEST here

**Pull request merged**